v2/stack_analyses.yaml

```diff
@@ -544,6 +546,11 @@ components:
       title: StackAnalysesResult
       type: object
       required:
+        - version
+        - started_at
+        - finished_at
```

Comment:
Can we rename this as "ended_at", as we use this attributes in the backbone flow. We can unify the naming of attributes. Also started_at looks perfect with ended_at